src/routes/datasets/[id].svelte

```diff
@@ -1,29 +1,91 @@
 <script context='module'>
+	import { writable } from 'svelte/store';
+	import {
+		aggregationsPerType,
+		determineESType,
+		buildAggregation
+	} from 'app/elasticsearch';
+	import {
+		request
+	} from 'app/net';
+
+	import JSONTree from 'svelte-json-tree'
+
+	import routes from 'app/data/routes.json';
+
+	export const aggStore = writable("");
```

Comment:
I'd move this in `app/stores.js` then importing it here

Reply:
I removed the store since it makes it easier to create multiple instances of the component at the same time as it doesn't rely on a 'singleton'. See ad0c48c